.github/workflows/lint.yml

```diff
@@ -37,3 +37,19 @@ jobs:
 
         - name: Run i18 Validation
           run: npm run i18n:validate
+
+  style-lint:
```

Comment:
I think we might be able to avoid having two linting jobs by changing the ESlint action to run one centralized `lint` script which would run both ESlint and Stylelint (which will be configured in `package.json`). That way we won't need a separate Stylelint job here

Reply:
for this we need to install a new package: https://www.npmjs.com/package/npm-run-all

Reply:
We could, if we wanted to. Otherwise i'm pretty sure we could add `&& npm run lint:stylelint` to the lint script to achieve the same purpose.

Reply:
It's installed :)